### PR TITLE
Add USDC and other tokens to examples. Update docs

### PR DESCRIPTION
**Pull request by rpedroni**
To get this working locally, do these 2 things:

1) In the SDK repo, change these lines in `App.tsx`
```tsx
const config = new Configuration({ network: 'local' });
// const config = new Configuration({ network: 'mainnet' });
```

2) In the BE repo, comment out these lines to make sure Uni's smart router works correctly for decent quote:
```yml
      - '0.0.0.0'
      - --fork-url
      - 'https://mainnet.infura.io/v3/cdf3bdd9227744688fb91b658146d51b'
      # - --fork-block-number
      # - '15939956'
      # - --block-time
      # - '5'
```

### CODE REVIEW

File: examples/src/sections/CharitableGiving.tsx [hunk 1]
```diff
@@ -38,47 +49,95 @@ const TOKENS = [
...
   const [selectedToken, setSelectedToken] = useState(TOKENS[0]);
   const [amountIn, setAmountIn] = useState('1');
-  const [exp, setExp] = useState(1e18);
+  const [amountInParsed, setAmountInParsed] = useState(parseUnits(amountIn || '0', selectedToken.decimals));
   const [loading, setLoading] = useState(false);
   const [swapAndDonateTransaction, setSwapAndDonateTransaction] = useState<NdaoSdkDonationSwap>();
-  const { config, error } = usePrepareSendTransaction({
+
+  // Send swap and donate transaction
+  const { config: config1 } = usePrepareSendTransaction({
```
Comment (Naguumo) [inline examples/src/sections/CharitableGiving.tsx]: I would like to see this be better named

File: examples/src/sections/CharitableGiving.tsx [hunk 2]
```diff
@@ -38,47 +49,95 @@ const TOKENS = [
...
+  const { data: entityAllowance } = useContractRead({
+    address: selectedToken.contractAddress as Address,
+    abi: erc20ABI,
+    functionName: 'allowance',
+    args: [address as Address, swapAndDonateTransaction?.to as Address],
+    enabled: !isETH && !!address && !!swapAndDonateTransaction?.to,
+    watch: true,
+  });
+
+  const { config: config2 } = usePrepareContractWrite({
```
Comment (Naguumo) [inline examples/src/sections/CharitableGiving.tsx]: I would like to see this be better named

**Review by Naguumo — DISMISSED** [review]

File: examples/src/sections/CharitableGiving.tsx [hunk 3]
```diff
@@ -27,58 +38,106 @@ const TOKENS = [
...
+    enabled: !isETH && !!address && !!swapAndDonateTransaction?.to,
+    overrides: {
+      gasLimit: BigNumber.from(100_000),
+    },
+  });
+  const { write: erc20Approve } = useContractWrite(config2);
+
+  const requiresAllowance = entityAllowance && entityAllowance.lt(amountInParsed);
+
+  console.log(amountIn, amountInParsed, entityAllowance, requiresAllowance);
```
Comment (Naguumo) [inline examples/src/sections/CharitableGiving.tsx]: Remove logs?

File: examples/src/sections/CharitableGiving.tsx [hunk 4]
```diff
@@ -27,58 +38,106 @@ const TOKENS = [
...
+    if (isNaN(+v)) return;
+    setSwapAndDonateTransaction(undefined);
+
+    setAmountIn(v);
+    setAmountInParsed(parseUnits(v || '0', selectedToken.decimals));
+  };
+
+  const handleApproveAllowance = () => {
+    if (requiresAllowance && erc20Approve) erc20Approve();
+  };
```
Comment (Naguumo) [inline examples/src/sections/CharitableGiving.tsx:126]: Attempting to donate USDC gives me an infinite loader

**Review by Naguumo — CHANGES_REQUESTED** [review]
This doesn't seem to be in a working condition for USDC donations

File: examples/src/sections/CharitableGiving.tsx (hunk 4 above)
Comment (rpedroni) [reply examples/src/sections/CharitableGiving.tsx:126]: Please see PR description (just updated)

**Review by Naguumo — APPROVED** [review]
Had to fix a bug with changing the selected token but otherwise LGTM!

**Pull request merged**